frontend/awx/administration/instance-groups/ContainerGroupForm.tsx

```diff
@@ -0,0 +1,209 @@
+import { useTranslation } from 'react-i18next';
+import {
+  LoadingPage,
+  PageFormCheckbox,
+  PageFormDataEditor,
+  PageFormSubmitHandler,
+  PageFormTextInput,
+  PageHeader,
+  PageLayout,
+  useGetPageUrl,
+  usePageNavigate,
+} from '../../../../framework';
+import { AwxRoute } from '../../main/AwxRoutes';
+import { AwxPageForm } from '../../common/AwxPageForm';
+import { InstanceGroup as ContainerGroup } from '../../interfaces/InstanceGroup';
+import { awxAPI } from '../../common/api/awx-utils';
+import { usePostRequest } from '../../../common/crud/usePostRequest';
+import { useGetItem } from '../../../common/crud/useGet';
+import { useParams } from 'react-router-dom';
+import { AwxError } from '../../common/AwxError';
+import { usePatchRequest } from '../../../common/crud/usePatchRequest';
+import { PageFormCredentialSelect } from '../../access/credentials/components/PageFormCredentialSelect';
+import { useFormContext, useWatch } from 'react-hook-form';
+import { PageFormSection } from '../../../../framework/PageForm/Utils/PageFormSection';
+import { SummaryFieldCredential } from '../../interfaces/summary-fields/summary-fields';
+import { POD_SPEC_DEFAULT_VALUE } from './constants';
```

Comment:
I believe this value comes from an OPTION request from the API. If we have this value provided by the API better to get from there in case the API is updated.